.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: build
+on: [pull_request, push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+           docker build -f py.Dockerfile \
+             --build-arg PYTHON_VERSION=${{ matrix.python-version }} \
+             --tag gym-docker .
+      
+      # TODO: Add and fix tests for pytest
```

Comment:
Do we want this to run the current `run_tests.py` file?

Reply:
We should probably wait until we make the environments working with the new gym v0.25.

Reply:
No worries